lib/Bio/EnsEMBL/DataCheck/Checks/SpeciesCommonName.pm

```diff
@@ -0,0 +1,77 @@
+=head1 LICENSE
+
+Copyright [2018-2021] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::SpeciesCommonName;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Data::Dumper;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME        => 'SpeciesCommonName',
+  DESCRIPTION => 'Meta key species.common_name should be same for group of strains or breed',
+  GROUPS      => ['core', 'meta'],
+  DB_TYPES    => ['core'],
+  TABLES      => ['meta'],
+};
+
+sub tests {
+  my ($self) = @_;
+  my $mca = $self->dba->get_adaptor("MetaContainer");
+  my $strain = $mca->single_value_by_key('species.strain');
```

Comment:
Don't need to get/check this value - the important one for this check is 'species.strain_group'.